### PR TITLE
feat(translation) enable cc translation

### PR DESCRIPTION
**Pull request by tamasdomokos**

### CODE REVIEW

File: react/features/subtitles/components/LanguageSelectorDialog.web.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+import LanguageList from './LanguageList';
+
+interface ILanguageSelectorDialogProps {
+    _language: string,
+    t: Function,
+}
+
+/**
+ * Component that renders the security options dialog.
```
Comment (damencho) [inline react/features/subtitles/components/LanguageSelectorDialog.web.tsx:22]: Ups :) Copy Paste I guess :)

Comment (tamasdomokos) [reply react/features/subtitles/components/LanguageSelectorDialog.web.tsx]: Ups, I guess :)

File: react/features/subtitles/components/LanguageListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+     * Click callback.
+     */
+    onLanguageSelected: (lang: string) => void;
+
+    /**
+     * The id of the button.
+     */
+    selected?: boolean;
+
+    t: Function,
```
Comment (dudumanbogdan) [inline react/features/subtitles/components/LanguageListItem.tsx:31]: I saw that we can use `extends WithTranslation {` and when we do this, than this line can be removed as t will be present by extending WithTranslation

**Comment by damencho** [comment]
@tamasdomokos can you fix the checks, the problem is that the translations are not sorted.

File: react/features/subtitles/components/LanguageListItem.tsx (hunk 2 above)
Comment (tamasdomokos) [reply react/features/subtitles/components/LanguageListItem.tsx]: done

**Comment by damencho** [comment]
Jenkins test this please

**Comment by damencho** [comment]
Jenkins add to whitelist

**Review by damencho — APPROVED** [review]

**Pull request merged**